CreeDictionary/API/models.py

```diff
@@ -179,7 +179,7 @@ def get_absolute_url(self) -> str:
             "cree-dictionary-index-with-lemma", kwargs={"lemma_text": self.text}
         )
         if self.homograph_disambiguator is not None:
-            lemma_url += f"?{self.homograph_disambiguator}={quote(getattr(self, self.homograph_disambiguator))}"
+            lemma_url += f"?{self.homograph_disambiguator}={quote(str(getattr(self, self.homograph_disambiguator)))}"
```

Comment:
situation arose where `self.homograph_disambiguator=="id"` in integration test. and `quote` errors out if the argument is an `int`

Reply:
I highly doubt this causes #490. So this should fix it.